api/v1/appchart_types.go

```diff
@@ -41,6 +41,28 @@ type AppChartSpec struct {
 
 	// HelmChart is the name of the Helm chart used to deploy an application.
 	HelmChart string `json:"helmChart,omitempty"`
+
+	// Values provides settings, i.e. field names and values to customize
+	// the referenced helm chart when deploying an application with this app
+	// chart. Note that user-configurable settings are declared with
+	// `Settings` instead. While nothing checks against exposing a field set
+	// here to the user this is strongly discouraged, to avoid confusion.
+	Values map[string]string `json:"values,omitempty"`
+
+	// Settings declares the fields underneath `userValues` the user is
+	// allowed to customize when deploying an application with the helm
+	// chart referenced by this app chart.
+	Settings map[string]AppChartSetting `json:"settings,omitempty"`
+
+	// To expand and clarify the above a bit more:
+	//
+	// - Values is the app chart configuring the helm chart. This enables the
+	//   creation of multiple app charts based on the same helm chart, as
+	//   predefined setups the user can select from.
+	//
+	// - Settings is telling the client(s) which config fields of the helm
+	//   chart the user is allowed to change. I.e. the knobs the user has to
+	//   tune the deployment with the given app chart.
```

Comment:
And here, instead of `user` maybe we should use the term `developer` or at least put it in parenthesis.

Reply:
Done dd4edb2dacd708c1efc6cc517cb5f67cffba4c76